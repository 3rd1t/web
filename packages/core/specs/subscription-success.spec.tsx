import React from "react"
import { render } from "@testing-library/react"

import SubscriptionSuccess from "../pages/subscribe/success"

describe("SubscriptionSuccess", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<SubscriptionSuccess />)
    expect(baseElement).toMatchSnapshot()
  })
})
