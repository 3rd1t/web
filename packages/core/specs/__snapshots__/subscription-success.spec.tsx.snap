// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscriptionSuccess should render successfully 1`] = `
<body>
  <div>
    <section>
      <div
        class="flex items-center min-h-screen bg-gray-100"
      >
        <div
          class="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
        >
          <div
            class="flex flex-col items-center justify-center"
          >
            <h1
              class="text-xl text-gray-900 md:text-2xl lg:text-3xl xl:text-4xl"
            >
              Thank you for subscribing, we will be in touch shortly.
            </h1>
            <button
              class="flex items-center mt-6 space-x-2 text-sm font-semibold hover:text-gray-900 text-research-600 focus:outline-none focus:underline"
            >
              <svg
                class="w-5 h-5 arrow-left"
                fill="none"
                stroke="currentColor"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 19l-7-7m0 0l7-7m-7 7h18"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
              <span>
                Go back
              </span>
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>
</body>
`;
